yid_langchain_extensions/llm/tools_in_prompt_llm.py

```
@@ -0,0 +1,195 @@
+import secrets
+import secrets
```

Ruff (0.8.2)

Redefinition of unused `secrets` from line 1

Remove definition: `secrets`

(F811)